app/javascript/components/widgets/numbers/falling-blocks.jsx

```diff
@@ -40,7 +40,15 @@ class Scene extends React.Component {
       Engine.merge(this.engine, { world: loadedWorld });
     }
     this.addBlocks();
-    this.freezeBlocks();
+    let i;
+    const allBodies = Composite.allBodies(this.engine.world);
+    if (allBodies.length > 1500) {
+      for (i = 3; i < allBodies.length; i += 1) {
```

Comment:
I assume we start at 3 to skip the walls? lets try and find a better way to identify and filter those out. I imagine it would look something like `allBodies.filter(isWall).map(...)`.